### ISSUE TITLE
Iron router

### ISSUE THREAD

**Issue by pciavald** [pciavald/blockfest#80]
Implémenter iron router pour pouvoir travailler sur base.blockfest.fr et team.blockfest.fr

**Comment by Xalava** [comment]
Yep,I will, par contre semaine histérique, je pourrai autour de vendredi

**Comment by pciavald** [comment]
@Xalava on est en train d'avancer sur base.blockfest, ça peut être en place pour quand ?
merci :)
